src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![doc = include_str!("../README.md")]
 #![deny(missing_debug_implementations)]
 #![deny(missing_docs)]
+#![deny(clippy::missing_inline_in_public_items)]
```

Comment:
I don't use clippy, nor do I believe this particular rule is useful to apply 100% of the time, so I don't think it makes sense to add this.